src/utils.ts

```diff
@@ -19,6 +30,51 @@ class Utils {
     );
   }
 
+  /**
+   * Encode the given classic address and tag into an x-address.
+   *
+   * @see https://xrpaddress.info/
+   *
+   * @param classicAddress A classic address to encode.
+   * @param tag An optional tag to encode.
+   * @returns A new x-address if inputs were valid, otherwise undefined.
+   */
+  public static encodeXAddress(
+    classicAddress: string,
+    tag: number | undefined
+  ): string | undefined {
+    if (!addressCodec.isValidClassicAddress(classicAddress)) {
+      return undefined;
+    }
+
+    // Xpring Common JS's API takes a string|undefined while the underlying address library wants string|false.
+    const shimTagParameter = tag != undefined ? tag : false;
+    return addressCodec.classicAddressToXAddress(
+      classicAddress,
+      shimTagParameter
+    );
+  }
+
+  /**
+   * Decode a `ClassicAddress` from a given x-address.
+   *
+   * @see https://xrpaddress.info/
+   *
+   * @param xAddress The xAddress to decode.
+   * @returns A `ClassicAddress`
+   */
+  public static decodeXAddress(xAddress: string): ClassicAddress | undefined {
+    if (!addressCodec.isValidXAddress(xAddress)) {
+      return undefined;
+    }
+
+    let shimClassicAddress = addressCodec.xAddressToClassicAddress(xAddress);
+    return {
+      address: shimClassicAddress.classicAddress,
+      tag: shimClassicAddress.tag != false ? shimClassicAddress.tag : undefined
```

Comment:
Is there a reason here to have "!=" instead of "!=="? If `tag === "0"` or `tag === 0` that is equivalent to false when using `!=`

Reply:
Done